t2.rb

```diff
@@ -0,0 +1,2 @@
+# bad
+[6, 1, 5].sort.first
```

Comment:
Use `min` instead of `sort...first`.

```suggestion
[6, 1, 5].min
```